### PR TITLE
Fix broken long option for ignore

### PR DESCRIPTION
**Pull request by ecdye**

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -168,7 +168,7 @@ int main(int argc, char *argv[]) {
     int long_index = 0;
     program_name = basename(argv[0]);
 
-    while ((opt = getopt_long_only(argc, argv, "l:u:m:L:U:hvbV", long_options, &long_index)) != -1) {
+    while ((opt = getopt_long_only(argc, argv, "l:u:m:L:U:ihvVb", long_options, &long_index)) != -1) {
```
Comment (UVV-gh) [inline main.c:171]: Oops, sorry for this regression :/

File: main.c [hunk 2]
```diff
@@ -57,7 +57,7 @@ void print_help(const char *program) {
     puts("  -m, --mountdir=MOUNTDIR    the mountdir of OverlayFS (optional)");
     puts("  -L, --lowernew=LOWERNEW    the lowerdir of new OverlayFS (optional)");
     puts("  -U, --uppernew=UPPERNEW    the upperdir of new OverlayFS (optional)");
-    puts("  -i  --ignore-mounted       don't prompt if OverlayFS is still mounted (optional)");
+    puts("  -i  --ignore               don't prompt if OverlayFS is still mounted (optional)");
```
Comment (UVV-gh) [inline main.c]: I believe the option name should be fixed at line 159 instead, so we keep `ignore-mounted` name.

**Review by UVV-gh — CHANGES_REQUESTED** [review]

File: main.c (hunk 1 above)
Comment (ecdye) [reply main.c:171]: No worries, it happens

File: main.c (hunk 2 above)
Comment (ecdye) [reply main.c]: Sure, I can make that change

**Review by UVV-gh — APPROVED** [review]

**Pull request merged**